docs/admin/node.md

```diff
@@ -165,6 +165,11 @@ completely unhealthy (i.e. there are no healthy nodes in the cluster). In such
 case, the node controller assumes that there's some problem with master
 connectivity and stops all evictions until some connectivity is restored.
 
+In Kubernetes 1.6, we introduced the logic for the node controller to handle
```

Comment:
But here https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/admin/node.md#node-controller we did have similar sentence as 

```
In Kubernetes 1.4, we updated the logic of the node controller
```

Shall we keep consistent?